File: library/src/main/java/com/pengrad/telegrambot/model/request/InlineQueryResultGame.java
package com.pengrad.telegrambot.model.request;

import java.io.Serializable;

/**
 * Stas Parshin
 * 03 October 2016
 */
public class InlineQueryResultGame extends InlineQueryResult<InlineQueryResultGame> implements Serializable {
    private final static long serialVersionUID = 0L;

    private String game_short_name;

    public InlineQueryResultGame(String id, String gameShortName) {
        super("game", id);
        game_short_name = gameShortName;
    }
}

File: core/src/main/kotlin/com/pengrad/telegrambot/model/request/InlineQueryResultGame.kt
package com.pengrad.telegrambot.model.request

import java.io.Serializable

/**
 * Stas Parshin
 * 03 October 2016
 */
class InlineQueryResultGame(id: String, private val game_short_name: String) : InlineQueryResult<InlineQueryResultGame>("game", id), Serializable {
    companion object {
        private const val serialVersionUID = 0L
    }
}
